tests/circuit_synthesis/test_encoder_synthesis.py

```
@@ -405,3 +409,57 @@ def test_local_minimum() -> None:
+    [
+        None,
+        {},
+        {0: "0"},
+        {0: "+"},
+        {1: "0", 0: "+"},
+        {0: "+", 2: "0"},
+        {0: "0", 1: "+", 2: "0"},
+    ],
+)
+def test_encoding_fixed_logical_qubits(
+    non_css_8_qubit: StabilizerCode, fixed_logical_qubits: dict[int, Literal["0", "+"]] | None, use_cnots_if_css: bool
```

Ruff (0.15.20)

[warning] Boolean-typed positional argument in function definition

(FBT001)